gemrb/core/Game.cpp

```diff
@@ -2130,9 +2130,7 @@ const Color *Game::GetGlobalTint() const
 		if (WeatherBits&WB_RAIN) {
 			return &DarkTint;
 		}
-		if (WeatherBits&WB_FOG) {
-			return &FogTint;
-		}
+		//TODO: add FogTint when it is fixed to not hinder lightness calculation
```

Comment:
One todo is enough if they're all equal. You can remove this one and the one before it. As we found out, tinting can't be used to get a fog effect, at least not the way we do it now. But it would probably look wonky either way: http://i.imgur.com/yBxaLO5.jpg

Something with light dithering would perhaps work, but that's for another time.